great_expectations/expectations/metrics/map_metric_provider.py

```diff
@@ -2326,7 +2298,7 @@ def _sqlalchemy_column_pair_map_condition_filtered_row_count(
     column_B_name = accessor_domain_kwargs["column_B"]
 
     column_names: List[Union[str, quoted_name]] = [column_A_name, column_B_name]
-    verify_column_names_exist(
+    _ = verify_column_names_exist(
```

Comment:
I though `verify_column_names_exist()` returned `None`. Is there a reason for the `_`?

Reply:
thought*

Reply:
@Shinnnyshinshin Brilliant catch!  Fixing (in next commit).  Thank you!